docs/docs/1.1.1/deployments/how-to-multi-zone.md

```diff
@@ -0,0 +1,311 @@
+
+### Cross-communication details
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+
+To view the list of service names available for cross-zone communication, run:
+
+```bash
+$ kubectl get dataplanes -n echo-example -o yaml | grep service
+           service: echo-server_echo-example_svc_1010
+```
+
+To consume the example service only within the same Kuma zone, you can run:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server:1010
+```
+
+To consume the example service across all zones in your Kuma deployment (that is, from endpoints ultimately connecting to the same global control plane), you can run either of:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh:80
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh:80
+```
+
+And if your HTTP clients take the standard default port 80, you can the port value and run either of:
+
+```bash
+<kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh
+<kuma-enabled-pod>$ curl http://echo-server.echo-example.svc.1010.mesh
+```
+
+Because Kuma on Kubernetes relies on transparent proxy, `kuma-dp` listens on port 80 for all virtual IPs that are assigned to services in the `.mesh` DNS zone. The DNS names are rendered RFC compatible by replacing underscores with dots.
+
+:::
+::: tab "Universal"
+
+With a hybrid deployment, running in both Kubernetes and Universal mode, the service tag should be the same in both environments (e.g `echo-server_echo-example_svc_1010`):
+
+```yaml
+type: Dataplane
+mesh: default
+name: backend-02 
+networking:
+  address: 127.0.0.1
+  inbound:
+  - port: 2010
+    servicePort: 1010
+    tags:
+      kuma.io/service: echo-server_echo-example_svc_1010
+```
+
+With a Universal control plane, the service tag has no such limitation.
+
+REVIEWER: DOES THIS ^ APPLY TO A HYBRID ENVIRONMENT WHERE THE CONTROL PLANE RUNS IN UNIVERSAL MODE? DOES THIS REFER TO THE GLOBAL CONTROL PLANE? CAN WE CLARIFY THE DISTINCTION BETWEEN THESE TWO CASES (THAT IS, THE HYBRID AND THIS SECOND CASE WHICH MIGHT OR MIGHT NOT BE RELEVANT TO THE FIRST ONE)?
+
+To consume a distributed service in a Universal deployment, where the application address is `http://localhost:20012`:
+
+```yaml
+type: Dataplane
+mesh: default
+name: web-02 
+networking:
+  address: 127.0.0.1
+  inbound:
+  - port: 10000
+    servicePort: 10001
+    tags:
+      kuma.io/service: web
+  outbound:
+  - port: 20012
+    tags:
+      kuma.io/service: echo-server_echo-example_svc_1010
+```
+
+:::
+::::
+
+The Kuma DNS service format (e.g. `echo-server_kuma-test_svc_1010.mesh`) is a composition of Kubernetes Service Name (`echo-server`),
+Namespace (`kuma-test`), a fixed string (`svc`), the service port (`1010`). The service is resolvable in the DNS zone `.mesh` where
+the Kuma DNS service is hooked.
+
+### Delete a zone
+
+REVIEWER: CANNOT TELL WHAT THE COMMANDS ARE THAT YOU RUN, AND IN WHAT ORDER. THIS SECTION NEEDS MUCH HELP, PLEASE!
+
```

Comment:
If the Global is K8s - delete the zone resource: something like `kubectl delete zone...." 
If Global is Universal using `kumactl delete zone ...`